qiskit/pulse/builder.py

```diff
@@ -2428,13 +2461,21 @@ def measure(
             registers = list(registers)
         except TypeError:
             registers = [registers]
-
-    measure_sched = macros.measure(
-        qubits=qubits,
-        inst_map=backend.defaults().instruction_schedule_map,
-        meas_map=backend.configuration().meas_map,
-        qubit_mem_slots={qubit: register.index for qubit, register in zip(qubits, registers)},
-    )
+    # backendV2
+    if hasattr(backend, "target"):
```

Comment:
I think you can just pass `backend` without branching, because `macros.measure` implicitly dispatches the scheduler logic for backends with different models.

Reply:
Fixed at https://github.com/Qiskit/qiskit-terra/pull/10150/commits/a21cc84d657c3cc5b9d1d6a8b0d22c733c8bae32.